app/components/nav-header.tsx

```diff
@@ -27,24 +27,24 @@ export function NavHeader() {
 				{appId && (
 					<div
 						className={clsx(
-							"flex text-sm font-mono border rounded-lg divide-x dark:(border-neutral-700 divide-neutral-700) overflow-hidden",
+							"flex text-sm font-mono border-2 rounded-3xl divide-x dark:(border-neutral-700 divide-neutral-700) overflow-hidden",
 							{
 								"!border-blue-500/70": isLogs && appId,
 								"!border-rose-500/70": isRequests && appId,
```

Comment:
remove the opacity from all the classes

```suggestion
								"!border-blue-500": isLogs && appId,
```